GNOME: more updates

jtojnar opened the pull request:
###### Description of changes

https://discourse.gnome.org/t/important-gssdp-gupnp-1-6-2/12394

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

jtojnar committed:
amtk: 5.6.0 → 5.6.1

https://gitlab.gnome.org/World/amtk/-/compare/5.6.0...5.6.1

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.gedit: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gedit/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome-builder: 43.2 → 43.3

https://gitlab.gnome.org/GNOME/gnome-builder/-/compare/43.2...43.3

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gssdp_1_6: 1.6.1 → 1.6.2

https://gitlab.gnome.org/GNOME/gssdp/-/compare/gssdp-1.6.1...gssdp-1.6.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gupnp_1_6: 1.6.1 → 1.6.2

https://gitlab.gnome.org/GNOME/gupnp/-/compare/gupnp-1.6.1...gupnp-1.6.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

fabianhjr reviewed (approved):
LGTM, building.

bobby285271 commented:
> building.

(Triage) Hi, how is the testing going? I went through the changes this PR is probably introducing  https://gitlab.gnome.org/GNOME/NetworkManager-openvpn/-/issues/112 (I don't know), but the other parts LGTM.

jtojnar committed:
evolution: 3.46.1 → 3.46.2

https://gitlab.gnome.org/GNOME/evolution/-/compare/3.46.1...3.46.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.gnome-initial-setup: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.gnome-maps: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gnome-maps/-/compare/v43.1...v43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.gnome-remote-desktop: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.gnome-software: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gnome-software/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.nautilus: 43.0 → 43.1

https://gitlab.gnome.org/GNOME/nautilus/-/compare/43.0...43.1

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome-builder: 43.3 → 43.4

https://gitlab.gnome.org/GNOME/gnome-builder/-/compare/43.3...43.4

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
libnma: 1.10.2 → 1.10.4

https://gitlab.gnome.org/GNOME/libnma/-/compare/1.10.2...1.10.4

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
libshumate: 1.0.2 → 1.0.3

https://gitlab.gnome.org/GNOME/libshumate/-/compare/1.0.2...1.0.3

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
mm-common: 1.0.4 → 1.0.5

https://gitlab.gnome.org/GNOME/mm-common/-/compare/1.0.4...1.0.5

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
evolution-data-server: 3.46.1 → 3.46.2

https://gitlab.gnome.org/GNOME/evolution-data-server/-/compare/3.46.1...3.46.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
evolution-ews: 3.46.1 → 3.46.2

https://gitlab.gnome.org/GNOME/evolution-ews/-/compare/3.46.1...3.46.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar commented:
Dropped that update and added more (starting with e-d-s).

jtojnar committed:
xdg-desktop-portal-gtk: 1.14.0 → 1.14.1

https://github.com/flatpak/xdg-desktop-portal-gtk/releases/tag/1.14.1

jtojnar committed:
vte: 0.70.1 → 0.70.2

https://gitlab.gnome.org/GNOME/vte/-/compare/0.70.1...0.70.2

jtojnar committed:
gnome.gucharmap: 15.0.1 → 15.0.2

https://gitlab.gnome.org/GNOME/gucharmap/-/compare/15.0.1...15.0.2

jtojnar committed:
deja-dup: 43.4 → 44.0

https://gitlab.gnome.org/World/deja-dup/-/compare/43.4...44.0

fabianhjr commented:
Will rebuild / retest tomorrow

jtojnar commented:
Looks like 43.2 was scheduled for yesterday so we might just as well wait a bit for shell updates & others.

jtojnar committed:
gnome.gnome-characters: 43.0 → 43.1

https://gitlab.gnome.org/GNOME/gnome-characters/-/compare/43.0...43.1

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
tracker-miners: 3.4.1 → 3.4.2

https://gitlab.gnome.org/GNOME/tracker-miners/-/compare/3.4.1...3.4.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

fabianhjr commented:
Build and works :3

Ty for the update

jtojnar committed:
gnome.gnome-shell: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/gnome-shell/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar committed:
gnome.mutter: 43.1 → 43.2

https://gitlab.gnome.org/GNOME/mutter/-/compare/43.1...43.2

Changelog-Reviewed-By: Jan Tojnar <jtojnar@gmail.com>

jtojnar commented:
Shell appears to work in VM too.

jtojnar commented:
Hmm, that should probably be dropped, I confused it with the edge resistance patch.
